tests/core/full_node/test_mempool.py

```diff
@@ -277,17 +282,17 @@ async def test_basic_mempool_manager(self, bt, two_nodes_mempool, wallet_a, self
             (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 0, MempoolInclusionStatus.PENDING),
             (ConditionOpcode.ASSERT_HEIGHT_RELATIVE, 1, MempoolInclusionStatus.PENDING),
             # the absolute height and seconds tests require fresh full nodes to
-            # run the test on. Right now, we just launch two simulations and all
-            # tests use the same ones. See comment at the two_nodes_mempool fixture
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 2, MempoolInclusionStatus.SUCCESS),
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 3, MempoolInclusionStatus.SUCCESS),
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 4, MempoolInclusionStatus.PENDING),
-            # (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 5, MempoolInclusionStatus.PENDING),
+            # run the test on. The fixture (two_nodes_mempool) creates 3 blocks,
+            # then condition_tester2 creates another 3 blocks
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 4, MempoolInclusionStatus.SUCCESS),
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 5, MempoolInclusionStatus.SUCCESS),
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 6, MempoolInclusionStatus.PENDING),
+            (ConditionOpcode.ASSERT_HEIGHT_ABSOLUTE, 7, MempoolInclusionStatus.PENDING),
             # genesis timestamp is 10000 and each block is 10 seconds
-            # (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, 10029, MempoolInclusionStatus.SUCCESS),
-            # (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, 10030, MempoolInclusionStatus.SUCCESS),
-            # (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, 10031, MempoolInclusionStatus.FAILED),
-            # (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, 10032, MempoolInclusionStatus.FAILED),
+            (ConditionOpcode.ASSERT_SECONDS_ABSOLUTE, 10049, MempoolInclusionStatus.SUCCESS),
```

Comment:
It's such a big jump from 7 to 10049 - why is the calculation so different from the 10029 it replaces?
(I understand why the low numbers changed).

Oh, right, these are seconds, and we are using 10-second intervals to quantize these steps. OK.

Sometimes it's nice to build up the tested value as an explanation (eg. `10000 + 49`), but I don't think that's needed here.

Reply:
the low numbers (4,5,6 and 7) are block *heights*. The large numbers (starting at 10000) are *seconds*. note that those tests are for different conditions